src/context/datasetStateSlice.ts

```diff
@@ -179,24 +156,70 @@ export const loadEWSDataset = async (
   return new Promise<TableData>(resolve => resolve(tableDataWithEWSConfig));
 };
 
+const fetchHDC = async (
```

Comment:
Let's add some comments for non-WFP people please :)
What's HDC? What's the expected data format?